bigquery-writer/bigquery-writer.py

```diff
@@ -19,7 +19,7 @@
 SUBSCRIBER_NAME = "bq"
 KEYSPACE_NAME_INCOMING = "ilisten"
 KEYSPACE_NAME_UNIQUE = "ulisten"
-APP_CREDENTIALS_FILE = "bigquery-credentials.json"
+APP_CREDENTIALS_FILE = os.environ['GOOGLE_APPLICATION_CREDENTIALS']
```

Comment:
will this fail if the env variable doesn't exist? Is this the preferred behaviour?